.travis.yml

```diff
@@ -33,12 +37,12 @@ after_success:
 - bash <(curl -s https://codecov.io/bash)
 deploy:
   - provider: script
-    script: make deploy
+    script: make -j -O deploy
```

Comment:
I don't think you should go unlimited.  My guess is that -j4 is probably a realistic limit.

Reply:
Switched to `-j4`, thanks.